core/modules/loader/DoList.h

```diff
@@ -0,0 +1,195 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2018 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ *
+ */
+#ifndef LSST_QSERV_LOADER_DOLIST_H_
+#define LSST_QSERV_LOADER_DOLIST_H_
+
+// system headers
+#include <chrono>
+#include <list>
+
+// Qserv headers
+#include "util/ThreadPool.h"
+
+namespace lsst {
+namespace qserv {
+namespace loader {
+
+
+class Central;
+
+
+class TimeOut {
+public:
+    using TimePoint = std::chrono::system_clock::time_point;
+    using Clock = std::chrono::system_clock;
+
+    TimeOut(std::chrono::milliseconds timeOut) : _timeOut(timeOut) {}
```

Comment:
Is implicit conversion construction (from `std::chrono::milliseconds` to `TimeOut`) expected here?